applications/DEMApplication/custom_utilities/calculate_global_physical_properties.h

```diff
@@ -112,44 +107,36 @@ class SphericElementGlobalPhysicsCalculator
             for (ElementsArrayType::iterator it = GetElementPartitionBegin(r_model_part, k); it != GetElementPartitionEnd(r_model_part, k); ++it){
                 max_values[k] = std::max(max_values[k], (it)->GetGeometry()[0].FastGetSolutionStepValue(r_variable));
                 elem_counter++;
-
-              }
-
-          }
+            }
+        }
 
         // getting the maximum between threads:
-
         for (int k = 0; k < OpenMPUtils::GetNumThreads(); k++){
             max_val = std::max(max_val, max_values[k]);
-          }
+        }
 
         return max_val;
-      }
+    }
 
       //***************************************************************************************************************
       //***************************************************************************************************************
 
-      double CalculateMinNodalVariable(ModelPart& r_model_part, const Variable<double>& r_variable)
-      {
+    double CalculateMinNodalVariable(ModelPart& r_model_part, const Variable<double>& r_variable) {
         ElementsArrayType& pElements = r_model_part.GetCommunicator().LocalMesh().Elements();
 
-        if (pElements.size() == 0){
-            KRATOS_THROW_ERROR(std::invalid_argument, "Cannot compute minimum of the required nodal variable. Empty model part. Could not compute the maximum of the required variable ", r_variable);
-          }
+        KRATOS_ERROR_IF(pElements.size() == 0) << "Cannot compute minimum of the required nodal variable. Empty model part. Could not compute the maximum of the required variable " << r_variable << std::endl;
 
         ElementsArrayType::iterator it_begin = pElements.ptr_begin();
 
-        if (!it_begin->GetGeometry()[0].SolutionStepsDataHas(r_variable)){
-            KRATOS_THROW_ERROR(std::invalid_argument, "Cannot compute minimum of the required nodal variable. Missing variable ", r_variable);
-          }
+        KRATOS_ERROR_IF(!it_begin->GetGeometry()[0].SolutionStepsDataHas(r_variable)) << "Cannot compute minimum of the required nodal variable. Missing variable " << r_variable << std::endl;
```

Comment:
Same here

Reply:
Fixed here and in a couple of places more